authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java

```diff
@@ -734,19 +735,18 @@ public Boolean onUserAdded(User user) throws AuthorizationPluginException {
 
   @Override
   public Boolean onUserRemoved(User user) throws AuthorizationPluginException {
-    VXUserList list = rangerClient.searchUser(ImmutableMap.of("name", user.name()));
-    if (list.getListSize() == 0) {
-      LOG.warn("The user({}) doesn't exist in the Ranger!", user);
+    Optional<Long> userId = getUserId(user.name());
```

Comment:
Since you have use `Optional`, I have you can:

return userId.map(id -> {
rangerClient.deleteUser(id)
return True;
).orElseGet(() -> {
Log.warn
 return false;
)

Reply:
Addressed.